API/Schemas/TodoSchema.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using Backend.API.Queries;
+using Backend.API.Services;
+
+namespace Backend.API.Schemas
+{
+    public class TodoSchema : GraphQL.Types.Schema
```

Comment:
Why do we have two schemas? Presuming we're only delivering one service to the frontend, I believe it's more reasonable to create a `RootSchema` and rather use queries for segregation.